tracex/extraction/prototype/prompts.py

```diff
@@ -53,14 +55,18 @@ def get_date():
             "content": "Please give me one date between 01/01/2020 and 01/09/2023.",
         }
     ]
-    country = client.chat.completions.create(model=u.MODEL, messages=message, max_tokens=50, temperature=0.5)
+    country = client.chat.completions.create(
```

Comment:
this variable should not be named "country" as this is the function to get a random date